dbschema/budget-record.esdl

```diff
@@ -0,0 +1,13 @@
+module default {
+  type BudgetRecord extending Project::ContextAware {
```

Comment:
Maybe `Budget::Record` instead?
```suggestion
module Budget {
  type Record extending Project::ContextAware {
```

Reply:
I was waffling on splitting these two into their separate files, but ultimately I think that's better than combining them if they are small.  _However_, since the use case in this instance is the complete dependency of one on the other (i.e. it can't exist by itself), I think your solution here is the best.  Nice one.